opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -13,12 +13,27 @@ use opentelemetry::{
 #[cfg(feature = "logs_level_enabled")]
 use opentelemetry::logs::Severity;
 
+use std::sync::atomic::AtomicBool;
 use std::{borrow::Cow, sync::Arc};
 
+use lazy_static::lazy_static;
```

Comment:
I think `once_cell` is used more predominantly in the codebase vs. `lazy_static`.
However, there is now support in the standard library via `std::sync::OnceLock` for this too, which would eliminate both dependencies, so please consider if that would be better.
e.g. https://stackoverflow.com/a/27826181/4954738

Reply:
Good point. Let me revise

Reply:
Ah I don't think we can use `std::sync::OnceLock` as our MSRV is still in 1.65 while the `OnceLock` is introcuded in `1.70`